src/common/entity/state_active.ts

```diff
@@ -37,7 +39,13 @@ export function stateActive(stateObj: HassEntity, state?: string): boolean {
       return compareState === "active";
     case "camera":
       return compareState === "streaming";
-    default:
-      return true;
   }
+
+  // Needs to be after the switch statement since some domains such as "alert"
+  // do not consider "off" an inactive state (as there is "idle" for example).
+  if (compareState === "off") {
```

Comment:
Are we sure the domains in the switch can't be off? Maybe safer to add a custom case `alert`?

Reply:
If we add domains to that IF statement the logic would change, e.g. switches in state "off" would then get a different color than before this PR (would have the same color as "on" then).

Reply:
I had checked the design preview again for the `entity-states` with this PR and all still looks correct there to me as it was before.

Reply:
if a `media_player` state is `off`, it would return active, as it is not `standby` `return compareState !== "standby"`?

Reply:
I see your point now. Missed that one when checking the design preview as there is still not enough contrast between "on" and "off" for the media players IMHO, but that is a different topic😉 .